### ISSUE TITLE
Fix "BroadcastChannel"

### ISSUE THREAD

**Issue by jcrang** [mswjs/jest-fixed-jsdom#18]
As of [MSW 2.6.0](https://github.com/mswjs/msw/releases/tag/v2.6.0), `BroadcastChannel` is required to be defined. 

Have polyfilled this myself with:

```js
const { BroadcastChannel } = require('node:worker_threads');

Object.defineProperties(globalThis, {
    BroadcastChannel: { value: BroadcastChannel }
});
```

**Comment by kettanaito** [comment]
Hi, @jcrang. Thanks for proposing this. 

Is `BroadcastChannel` truly undefined in JSDOM?

**Comment by kettanaito** [comment]
We can also take it from the global scope, it's defined there.

**Comment by jcrang** [comment]
Thanks @kettanaito.

To answer your question (hopefully?) we were getting the same errors as for other properties that are fixed in `jest-fixed-jsdom`. The full error from our tests runs is:

```
  ● Test suite failed to run

    ReferenceError: BroadcastChannel is not defined
```

**Comment by kettanaito** [comment]
## Released: v0.0.7 🎉

This has been released in v0.0.7!

- 📄 [**Release notes**](https://github.com/mswjs/jest-fixed-jsdom/releases/tag/v0.0.7)
- 📦 [npm package](https://www.npmjs.com/package/jest-fixed-jsdom/v/0.0.7)

Make sure to always update to the latest version (`npm i jest-fixed-jsdom@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
